### PR TITLE
Update editor and MJML style

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This brings the editor closer to what the email will actually look like. There are still a few differences (e.g. padding on blocks with background colour) but these are pretty minor differences.

### How to test the changes in this Pull Request:

1. Create a new newsletter. Send a test email. Spot the differences.
2. Switch to this branch.
3. Refresh editor and send another test email. There should be less differences.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: includes/class-newspack-newsletters-renderer.php [hunk 1]
```diff
@@ -145,7 +145,7 @@ function ( $key ) use ( &$attrs ) {
 		);
 
 		if ( isset( $attrs['background-color'] ) ) {
-			$attrs['padding'] = '16px';
+			$attrs['padding'] = '0';
```
Comment (adekbadek) [inline includes/class-newspack-newsletters-renderer.php:148]: This makes background-colored paragraphs in columns render incorecctly:

image

Comment (thomasguillot) [reply includes/class-newspack-newsletters-renderer.php:148]: Should be fixed in https://github.com/Automattic/newspack-newsletters/pull/149/commits/3b3b1cda698db21bcb7106f4beaf4bac8ff8db7f

Comment (adekbadek) [reply includes/class-newspack-newsletters-renderer.php:148]: One more thing I forgot about earlier - same issue is with Heading block in a column.

Comment (thomasguillot) [reply includes/class-newspack-newsletters-renderer.php:148]: Oh yeah. I've been less specific in https://github.com/Automattic/newspack-newsletters/pull/149/commits/d2a5bdd1e3b767c102e000333e595d951159e95f, hopefully it should do the trick.

**Pull request merged**